catalog/spark/spark.bom

```diff
@@ -154,6 +164,9 @@ brooklyn.catalog:
           type: spark-node
           name: Spark Master
           id: spark-master
+          brooklyn.config:
+            spark.master.rest.url: $brooklyn:entity("spark-master").attributeWhenReady("spark.rest.url")
```

Comment:
This _is_ `spark-master`. You can just write `$brooklyn:attributeWhenReady("spark.rest.url")`.